hw/dv/sv/mem_bkdr_util/mem_bkdr_util.core

```diff
@@ -17,6 +17,8 @@ filesets:
       - sram_scrambler_pkg.sv
       - mem_bkdr_util_pkg.sv
       - mem_bkdr_util.sv: {is_include_file: true}
+      - sram_mem_bkdr_util.sv: {is_include_file: true}
```

Comment:
Prefer to use this naming convention for auxiliary files that are added to segregate methods that belong to the same base file / class: `mem_bkdr_util__sram.sv` etc. We have an example of this in CIP base vseq.

Reply:
Thank you for pointing that out! Updated.